src/test/kotlin/no/liflig/properties/PropertiesLoaderTest.kt

```diff
@@ -0,0 +1,83 @@
+package no.liflig.properties
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import java.util.Properties
+
+// By default no properties files will match the default names and be loaded
+class PropertiesLoaderTest {
+
+    @Test
+    fun `should load properties for normal runtime with SSM params`() {
+        val awsPath = "/construct/current"
+        val mockProperties = Properties().apply {
+            put("hacker.name", "Henrik")
+        }
```

Comment:
```suggestion
        val mockProperties = mapOf("hacker.name" to "Henrik").toProperties()
```

(vurder å inline så det blir likt den andre testen)